importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/LongTagImportingLineConsumer.java

```diff
@@ -29,23 +36,32 @@
 import org.itsallcode.openfasttrace.api.importer.input.InputFile;
 
 // [impl->dsn~import.full-coverage-tag~1]
+// [impl->dsn~import.full-coverage-tag-with-needed-coverage~1]
 class LongTagImportingLineConsumer extends RegexLineConsumer
 {
     private static final Logger LOG = Logger
             .getLogger(LongTagImportingLineConsumer.class.getName());
 
     private static final String COVERING_ARTIFACT_TYPE_PATTERN = "\\p{Alpha}+";
+    private static final String OPTIONAL_WHITESPACE = "\\s*";
     private static final String TAG_PREFIX = "\\[";
     private static final String TAG_SUFFIX = "\\]";
-    private static final String TAG_REGEX = TAG_PREFIX //
+    private static final String NEEDS_COVERAGE = ">>" + OPTIONAL_WHITESPACE + "([\\p{Alpha},\\s]+)";
```

Comment:
```suggestion
    private static final String NEEDS_COVERAGE = ">>" + OPTIONAL_WHITESPACE + "(\\p{Alpha}+(?:" +OPTIONAL_WHITESPACE + "," + OPTIONAL_WHITESPACE +  "\\p{Alpha}+)*)";
```

Please check this in the test. Writing a complex regex without IDE is error prone.